pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -924,6 +932,11 @@ func GetOpenshiftBootstrapClusterRoleBindings() []rbac.ClusterRoleBinding {
 		newOriginClusterBinding(MasterRoleBindingName, MasterRoleName).
 			Groups(MastersGroup).
 			BindingOrDie(),
+		// Everyone should be able to add a scope to their impersonation request.  It is purely tightening.
+		// This does not grant access to impersonate in general, only tighten if you already have permission.
+		newOriginClusterBinding(ScopeImpersonationRoleName, ScopeImpersonationRoleName).
```

Comment:
Just use `rbac.NewClusterBinding` directly since do not need the origin wrapper.

Reply:
> Just use rbac.NewClusterBinding directly since do not need the origin wrapper.

This just matches the others around it

Reply:
We already use `rbac.NewClusterBinding` in the places where the role and binding have the same name (there is at least one like that near the bottom).